app.py

```diff
@@ -42,7 +31,11 @@ def get(self):
 
     def post(self):
         content = request.json
-        validation.post_validation(content)
+        schema = RestaurantSchemaPost()
+        try:
+            schema.load(content)
+        except ValidationError as err:
```

Comment:
it is possible to use Flask error handler, just add error handler for Marshmallow `ValidationError` class

Reply:
and try... catch is not needed then

Reply:
fixed